borg/upgrader.py

```diff
@@ -139,12 +139,14 @@ def convert_cache(self, dryrun):
           `Cache.open()`, edit in place and then `Cache.close()` to
           make sure we have locking right
         """
-        caches = []
         transaction_id = self.get_index_transaction_id()
         if transaction_id is None:
             logger.warning('no index file found for repository %s' % self.path)
         else:
-            caches += [os.path.join(self.path, 'index.%d' % transaction_id).encode('utf-8')]
+            cache = os.path.join(self.path, 'index.%d' % transaction_id).encode('utf-8')
```

Comment:
this was already present in the upgrader code and not part of this patch.

but okay, i can rewire this.